svthalia/Reaxit
# Fix previously filtered events not showing on the calendar

Closes #468.

### Summary
Previously, a mistake in interpreting the dart syntax led to previous events being cleared before using them.

### How to test
Steps to test the changes you made:
1. Go to calendar
2. Scroll down to the bottom
3. See study trip
